eventmesh-common/src/test/resources/configuration.properties

```diff
@@ -23,4 +23,6 @@ eventMesh.server.name=value5
 eventMesh.server.hostIp=value6
 eventMesh.connector.plugin.type=rocketmq
 eventMesh.security.plugin.type=acl
-eventMesh.registry.plugin.type=namesrv
\ No newline at end of file
+eventMesh.registry.plugin.type=namesrv
+eventMesh.exporter.zipkin.ip=localhost
```

Comment:
keep the path the same as eventmesh.trace.export.zipkin.ip.

Reply:
fixed